src/NServiceBus.AcceptanceTesting/Support/EndpointRunner.cs

```diff
@@ -70,6 +77,14 @@ public async Task Initialize(RunDescriptor run, EndpointBehavior endpointBehavio
         }
     }
 
+    public class TbdTask : FeatureStartupTask
```

Comment:
Just to get it working, if we go with this approach a new overload to RegisterStartupTask could be added to handle this better